core/utils.js

```diff
@@ -281,22 +281,58 @@ Blockly.getRelativeXY_ = function(element) {
 };
 
 /**
- * Return the absolute coordinates of the top-left corner of this element.
- * The origin (0,0) is the top-left corner of the nearest SVG.
+ * Return true if child is descendant of parent element.
+ * @param {!Element} child Reference element.
+ * @param {!Element} parent Element to search in the parents of child.
+ * @return {!Boolean} Is descendant or not.
+ * @private
+ */
+Blockly.isDescendant_ = function(child, parent) {
```

Comment:
Why not just use goog.dom.contains(parent, descendant)?
https://closure-library.googlecode.com/git-history/docs/local_closure_goog_dom_dom.js.source.html#line1312